events/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Events
+description: The hub of all OLS events
+image: https://images.unsplash.com/photo-1535016120720-40c646be5580?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1740&q=80
+photos:
+  name: Alex Litvin
+  url: https://unsplash.com/photos/MAYsdoYpGuk
+---
+
+{% for event in site.data.events %}
```

Comment:
It might be better as a table a bit similar to : 

<img width="1134" alt="Screenshot 2024-03-21 at 19 39 18" src="https://github.com/open-life-science/open-life-science.github.io/assets/1842467/e6f32eb6-a7c3-43db-b7d5-1ce161462c20">

We could have all events in one table and just have a "organizer" column with OLS logo when it is organized by OLS